on_receive_fields.lua

```diff
@@ -1,7 +1,6 @@
 local S = minetest.get_translator("travelnet")
 
-
-function travelnet.on_receive_fields(pos, _, fields, player)
+local function on_receive_fields_internal(pos, _, fields, player)
```

Comment:
Just drop underscore arg _ from private function or Is there reason to keep it around? Should be enough to have it just for public travelnet.on_receive_fields function where it can be simply ignored.